src/components/Modals/FiatRamps/views/Manager.tsx

```diff
@@ -88,6 +90,20 @@ const ManagerRouter: React.FC<ManagerRouterProps> = ({ fiatRampProvider }) => {
   const {
     state: { wallet },
   } = useWallet()
+  const accountIds = useAppSelector(state =>
+    selectPortfolioAccountIdsByAssetId(state, { assetId: selectedAsset?.assetId ?? '' }),
+  )
+  const [accountId, setAccountId] = useState<AccountId | null>(null)
+
+  useEffect(() => {
+    if (!accountIds.length) return
+    setAccountId(accountIds[0])
+  }, [accountIds])
+
+  const handleAccountIdChange = useCallback((accountId: AccountId) => {
+    if (!accountId) return
```

Comment:
Missed this one on the initial review, but this is guaranteed to fire with an `AccountId`, revert this
```suggestion
```